src/app/components/bibliography/bibliography-item/bibliography-item.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { BibliographicCitation } from 'src/app/services/xml-parsers/bibliography-parser.service';
+
+@Component({
+  selector: 'evt-bibliography-item',
+  templateUrl: './bibliography-item.component.html',
+  styleUrls: ['./bibliography-item.component.scss'],
+})
+export class BibliographyItemComponent implements OnInit {
+  @Input() havingFields: boolean;
+  @Input() bibliographicCitation: BibliographicCitation;
+  public templateBiblCit;
+  ngOnInit() {
+    if (this.havingFields) {
+      this.templateBiblCit = {};
+      const values = Object.values(this.bibliographicCitation);
+      const keys = Object.keys(this.bibliographicCitation);
+      keys.forEach((k, i) => this.templateBiblCit[k] = values[i]);
```

Comment:
I totally agree with you. I used `templateBiblCit` because the `npm run build: gh-pages` build command complained about this. Now, with the move to Angular 9, the error already appears with `npm run start`. I want to use `bibligraphicCitation` directly, but compilers don't allow me to. Any suggestion?